### PR TITLE
fixes in structure tool and bumped sdk versions

### PR DESCRIPTION
**Pull request by muhammad-ali-e**
## What

- Updated/Fixed structure tool ni accordance with latest SDK
- Bumped SDK version in filesystem package and tools

## Why

- Need to be alligned with latest SDK

## How

-

## Can this PR break any existing features. If yes, please list possible items. If no, please explain why. (PS: Admins do not merge the PR without this section filled)

- No, under flags

## Database Migrations

- 

## Env Config

- 

## Relevant Docs

-

## Related Issues or PRs

-

## Dependencies Versions

-

## Notes on Testing

-

## Checklist

I have read and understood the [Contribution Guidelines]().

### CODE REVIEW

File: tools/classifier/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # Add your dependencies here
 
 # Required for all unstract tools
-unstract-sdk~=0.53.1
+unstract-sdk~=0.54.0rc2
```
Comment (chandrasekharan-zipstack) [inline tools/classifier/requirements.txt:4]: @muhammad-ali-e shouldn't this be rc5? That's the latest

File: tools/structure/requirements.txt
Comment (chandrasekharan-zipstack) [inline tools/structure/requirements.txt:1]: @muhammad-ali-e for all the tools, can you update the tool version as well correctly in its `config/properties.json`?

File: tools/classifier/requirements.txt (hunk 1 above)
Comment (gaya3-zipstack) [reply tools/classifier/requirements.txt:4]: @muhammad-ali-e Let us make it rc5 everywhere to be consistent.

Comment (muhammad-ali-e) [reply tools/classifier/requirements.txt:4]: it will take the latest rc due to `~=`

Comment (muhammad-ali-e) [reply tools/classifier/requirements.txt]: updated to rc5

**Review by chandrasekharan-zipstack — APPROVED** [review]
LGTM - looks like tool versions were updated previously but not released earlier. Hence `main` will contain the expected tool versions once this PR gets merged

**Review by gaya3-zipstack — APPROVED** [review]
Looks good.

**Pull request merged**